Sources/AppStoreConnectCLI/Commands/Certificates/ReadDownloadCertificateCommand.swift

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import ArgumentParser
+import Combine
+import Foundation
+import Files
+
+struct ReadDownloadCertificateCommand: CommonParsableCommand {
+    static var configuration = CommandConfiguration(
+    commandName: "read",
+    abstract: "Get information about a certificate and download the certificate data.")
+
+    @OptionGroup()
+    var common: CommonOptions
+
+    @Argument(help: "The certificate’s serial number. (eg. 1A23BCDEF4G5D6C7)")
+    var serial: String
+
+    @Option(help: "The file download path and name. (eg. ./file.cer)")
+    var certificateOutput: String?
+
+    func run() throws {
+        let service = try makeService()
+
+        let certificate = try service
+            .readCertificate(serial: serial)
+            .await()
```

Comment:
We could choose to put await in the service function here

Reply:
Is the service intended to be blocking @adamjcampbell? I can see reasons for both sides of this argument. Ie, service could be async vs entirely blocking (for simplicity).

Reply:
I've come down on the side of blocking for simplicity inside the service we can decide what is parallel and what is not